### ISSUE TITLE
Want simple callback function 

### ISSUE THREAD

**Issue by MOMMH2008** [kstenerud/KSCrash#166]
when crash detected call my function and give it the crash log ,
In my function I send the crash log to my server side through my API

i don't need all of that 

 KSCrashInstallation* installation = [self makeEmailInstallation]; 
KSCrashInstallation* installation = [self makeHockeyInstallation];
 KSCrashInstallation* installation = [self makeQuincyInstallation];
KSCrashInstallation *installation = [self makeVictoryInstallation];

the question is can i do that just call back function when detect crash and to be static like "on crash" ?

**Comment by Meonardo** [comment]
Yes, you can upload crash logs to your server. Using "KSCrashInstallationStandard" class and pass an URL.

**Comment by MOMMH2008** [comment]
yes but who can send my json format to my server , if you have example please support me

**Comment by naftaly** [comment]
Callback functions are available via the KSCrashConfiguration's onCrash callback property.
